src/core/server/elasticsearch/version_check/ensure_es_version.test.ts

```diff
@@ -122,13 +122,13 @@ describe('pollEsNodesVersion', () => {
     internalClient.nodes.info.mockImplementationOnce(() => createEsError(error));
   };
 
-  it('returns iscCompatible=false and keeps polling when a poll request throws', (done) => {
+  it('returns isCompatible=false and keeps polling when a poll request throws', (done) => {
     expect.assertions(3);
     const expectedCompatibilityResults = [false, false, true];
     jest.clearAllMocks();
 
     nodeInfosSuccessOnce(createNodes('5.1.0', '5.2.0', '5.0.0'));
-    nodeInfosErrorOnce('mock request error');
+    nodeInfosErrorOnce(new Error('mock request error'));
```

Comment:
NIT: it's a good idea to test an error that is just a rejected string (it _could_ technically happen)